lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -406,6 +407,8 @@ import qualified Cardano.Wallet.Primitive.Types.Coin as W
 import qualified Cardano.Wallet.Primitive.Types.Hash as W
 import qualified Cardano.Wallet.Primitive.Types.Redeemer as W
 import qualified Cardano.Wallet.Primitive.Types.RewardAccount as W
+import Cardano.Wallet.Primitive.Types.TokenBundle
+    ( TokenBundleSizeAssessment (..), TokenBundleSizeAssessor (..) )
```

Comment:
Hi @paolino

According to our coding standards, [imports should be grouped into three groups, separated by a blank newline](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md#stylish-haskell-is-used-to-format-grouped-imports--language-pragmas):

Prelude import
Explicit imports
Qualified imports

So explicit imports like `tokenBundleSizeAssessment` should be moved to the section above.